src/dashboard.js

```diff
@@ -16,13 +16,20 @@ __webpack_nonce__ = getCSPNonce() // eslint-disable-line
 __webpack_public_path__ = linkTo('integration_mastodon', 'js/') // eslint-disable-line
 
 document.addEventListener('DOMContentLoaded', () => {
+	if (!OCA.Dashboard) {
+		return
+	}
```

Comment:
maybe it can be console logged? If there are no other indications of this failing, that is.